lib/widgets/image_widget_pokemon_details.dart

```diff
@@ -0,0 +1,31 @@
+import 'package:flutter/material.dart';
+import 'package:pokedexbootcamp/utils/constants.dart';
+import 'package:pokedexbootcamp/utils/theme.dart';
+
+class PokemonDetailsImages extends StatelessWidget {
+  const PokemonDetailsImages({
+    Key? key,
+    this.pokeImage,
+  }) : super(key: key);
+
+  final dynamic pokeImage;
```

Comment:
why dynamic?

Reply:
The dynamic variable is for a way to pass the sprite images to the image network.